test/e2e.method.send.js

```diff
@@ -0,0 +1,199 @@
+var assert = require('assert');
+var Web3 = require('../packages/web3');
+var Basic = require('./sources/Basic');
+var utils = require('./helpers/test.utils');
+
+describe('method.send [ @E2E ]', function() {
+    var web3;
+    var accounts;
+    var basic;
+    var instance;
+    var options;
+
+    var basicOptions = {
+        data: Basic.bytecode,
+        gasPrice: '1',
+        gas: 4000000
+    };
+
+    describe('http', function() {
+        before(async function(){
+            web3 = new Web3('http://localhost:8545');
+            accounts = await web3.eth.getAccounts();
+
+            basic = new web3.eth.Contract(Basic.abi, basicOptions);
+            instance = await basic.deploy().send({from: accounts[0]});
+        })
+
+        it('returns a receipt', async function(){
+            var receipt = await instance
+                .methods
+                .setValue('1')
+                .send({from: accounts[0]});
+
+            assert(receipt.status === true);
+            assert(web3.utils.isHexStrict(receipt.transactionHash));
+        });
+
+        it('errors on OOG', async function(){
+            try {
+                await instance
+                    .methods
+                    .setValue('1')
+                    .send({from: accounts[0], gas: 100});
+
+                assert.fail();
+
```

Comment:
Such breaks aren't required:)

Reply:
Left a comment about this above in an unresolved note..